pytodo/modules.py

```diff
@@ -31,6 +31,7 @@ def add_todo(todo_text, due_date):
         "is_done": False,
         "created": datetime.now(),
         "due": parse_date(due_date),
+        "quotes_disabled": False,
```

Comment:
I see that you are assigning a new property for every todo object. It doesn't make sense to do it for every object, even though it will work. Imagine having a lot of todos, it will have to set the property for every object according to your code, which will take a lot of time. We are trying to keep the app simple, so this is not the best approach I think.

I would suggest creating a new collection called `config` in the same database (initialize the collection after line 9) with a single object that will contain the configuration for the app. That way, we can add any other customization for the entire app later.

Reply:
So the Config collection should have an element as "quotes" which will be created and a value will be inserted in the command line to the collection. Only the updation will be in modules.py
That's what you are suggesting?

Reply:
Yes, exactly. We can add some other configurations if we need them later. Right now, the collection will contain only one element.